scanrepository/scanrepository.go

```diff
@@ -40,7 +40,7 @@ type ScanRepositoryCmd struct {
 	// Determines whether to open a pull request for each vulnerability fix or to aggregate all fixes into one pull request
 	aggregateFixes bool
 	// The current project technology
-	projectTech coreutils.Technology
+	projectTech []coreutils.Technology
```

Comment:
A Project represents a technology with its working dirs, installCommand, etc. So this change is unnecessary.